src/main/java/com/openlattice/datastore/services/PostgresEntitySetManager.java

```diff
@@ -112,4 +120,42 @@ public EntitySet getEntitySet( String entitySetName ) {
         }
     }
 
+    public Set<UUID> getAllPropertyTypeIds() {
```

Comment:
Hazelcast is a more efficient way of gett all property type ids in a set. In fact it's a one liner in the edmService. ```propertyTypes.keySet()```

Reply:
Also you could have used StreamUtil to collect directly into a set instead of manually managing with try-with-resources but that is pretty minor.